### ISSUE TITLE
Asset Browser - Thumbnails aren't centered

### ISSUE THREAD

**Issue by KaesarZZ** [Facepunch/sbox-issues#4504]
### Describe the bug

I've made some changes in the skeleton of my playermodels and now they don't use ScaleandMirror and the thumbnails are seen from far away and make the model extremely small for some reason.

https://i.gyazo.com/63d4ff92dfec9f5d60cb4d303ade3fcc.png

https://asset.party/sboxmp/hazmatsuit

- I've tried "Regenerate Thumbnail" but it doesn't work.
- The model is correctly scaled ingame.
- It looks correctly in Preview.

### To Reproduce

Basically forcing a scale? but yeah it's selfexplained.

### Expected behavior

It should look centered as in the Preview.

https://i.gyazo.com/1ed048e0374d5fea96664a50d17495b0.png

**Comment by garrynewman** [comment]
This is fixed on the staging branch
